src/index.ts

```diff
@@ -1,3 +1,6 @@
+import parserJsonc from "jsonc-eslint-parser";
```

Comment:
[Bug] When I try this in a CommonJS config file (`eslint package.json --config eslint.config.cjs`), the default-imported module is `undefined`. `jsonc-eslint-parser` doesn't actually provide a `default` export.

```suggestion
import * as parserJsonc from "jsonc-eslint-parser";
```